nca/CoreDS/Peer.py

```diff
@@ -646,6 +646,21 @@ def rep(self):
         for peer in self:
             return str(peer)
 
+    def __str__(self):
+        res = ''
+        if len(self) > 1:
+            res += '{'
+        first = True
+        for peer in self:
+            if first:
+                first = False
+            else:
+                res += ', '
+            res += str(peer)
+        if len(self) > 1:
+            res += '}'
+        return res
+
```

Comment:
why not having `{`,`}`, for the case of a single peer? it is still a set..

Reply:
Thought it would be more beautiful. But you're right, `{ , }` is more consistent. Changed.